CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing to leaderboard-backend
+We appreciate your help!
+
+## Before filing an issue
+If you are unsure whether you have found a bug, please consider asking in [our discord](https://discord.gg/TZvfau25Vb) first.
+
+Similarly, if you have a question about a potential feature, [the discord](https://discord.gg/TZvfau25Vb) can be a fantastic resource for first comments.
+
+## Filing issues
+Filing issues is as simple as going to [the issue tracker](https://github.com/speedrun-website/leaderboard-backend/issues), and adding an issue using one of the below templates.
+
+### Feature Request / Task
+```
+{Feature Request/Task}: {short description}
+---
+{Detailed description}
+
+### Affected Functionality
+{Any known functionality impacted, or Unknown if further research needs to be done}
+
+### Other Relevant Issues
+{Links to Relevant Issues}
+```
+
+### Bugs
+```
+Bug: {short description}
+---
+{Summary of bug}
+
+### Step(s) to Reproduce
+{Numbered list of step(s)}
+
+### Expected Result
+{Summary of expected result}
+
+### Actual Outcome
+{Description of actual outcome}
+```
+
+## Contributing code
+### Example code contribution flow
+1. Make a fork of this repo.
+1. Name a branch on your fork something descriptive for this change (eg. `UpdateMakefile`).
+1. Make your changes.
+1. Include unit tests for all changes, and integration tests if it is appropriate (i.e. new endpoints are added).
+1. Verify your changes pass all existing and new tests. 
+1. Commit your changes (Tip! Please read our [Style guide](#style-guide) to help the pull request process go smoothly).
+1. Push your branch.
+1. Open a pull request to `leaderboardsgg/leaderboard-backend-poc`.
```

Comment:
A nudge to leave a proper pull request like "Don't forget to explain the context of your pull request.". A lot of folks open PRs and feel their code self-documents. Might be too hand holdy to include a specific call out though.